modules/auxiliary/gather/argus_dvr_4_lfi_cve_2018_15745.rb

```diff
@@ -0,0 +1,72 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+    include Msf::Exploit::Remote::HttpClient
+    include Msf::Auxiliary::Report
+
+def initialize(info = {})
+    super(
+        update_info(
+            info,
+            'Name' => 'Argus Surveillance DVR 4.0.0.0 - Directory Traversal',
+            'Description' => %q{
+                This module leverages an anauthenticated arbitrary file read for
+                the Argus Surveillance 4.0.0.0 system which never saw an update since.
+                As this is a Windows related application we recommend looking for common
+                Windows file locations, especially C:\ProgramData\PY_Software\Argus Surveillance DVR\DVRParams.ini
+                which houses another vulnerability in the Argus Surveillance system. This directory traversal vuln
+                is being tracked as CVE-2018-15745
+            },
+            'Author' => [ 'Maxwell Francis' ],
```

Comment:
We usually try to credit the module author as well as the person who discovered the vulnerability and or the person who wrote the PoC the module was based off of. 
```suggestion
            'Author' => [ 
              'Maxwell Francis', # msf module
              'John Page' # (aka hyp3rlinx) PoC 
             ],
```